pandas/core/indexes/datetimes.py

```diff
@@ -1753,7 +1753,32 @@ def freq(self, value):
     is_leap_year = _field_accessor(
         'is_leap_year',
         'is_leap_year',
-        "Logical indicating if the date belongs to a leap year")
+        """
+        Return a boolean indicating if the date belongs to a leap year.
+
+        A leap year is a year, occurring every four years, which has 366 days
```

Comment:
There are exceptions for this four year rule, years divisible by 100 are not leap years except if they are also divisible by 400.
From wiki:
   "Every year that is exactly divisible by four is a leap year, except for years that are exactly divisible by 100, but these centurial years are leap years if they are exactly divisible by 400. For example, the years 1700, 1800, and 1900 were not leap years, but the years 1600 and 2000 were."